src/Sql/Sql/Auditing/Cmdlet/GetAzSqlDatabaseAuditing.cs

```diff
@@ -19,6 +19,7 @@
 namespace Microsoft.Azure.Commands.Sql.Auditing.Cmdlet
 {
     [CmdletDeprecation(ReplacementCmdletName = VerbsCommon.Get +
+        "-" +
```

Comment:
need to add a change in the changelog so this is picked up in the next release.

Reply:
Done